packages/react/src/components/Attachment/Attachment.tsx

```diff
@@ -122,13 +122,10 @@ class Attachment extends UIComponent<ReactProps<AttachmentProps>, AttachmentStat
             })}
           </div>
         )}
-        {action && (
-          <div className={classes.action}>
-            {Button.create(action, {
-              defaultProps: { iconOnly: true, text: true },
-            })}
-          </div>
-        )}
+        {action &&
+          Button.create(action, {
```

Comment:
not sure about these changes in the elements structure provided to Attachment's layout, as those making things inconsistent - would expect either every slot being wrapped with div, or all not being wrapped. 

Also, given that styles for `action` slot are removed (https://github.com/stardust-ui/react/pull/1269/files?file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#diff-8bdbadd856993b56fb3e32cd80f44335L37), it seems questionable that `classes.action` originally applied to wrapper now is applied as `styles` to Button. 

Could you, please, suggest the reasons for these changes?

Reply:
> not sure about these changes in the elements structure provided to Attachment's layout, as those making things inconsistent - would expect either every slot being wrapped with div, or all not being wrapped.

good point, I was able to bring consistency by removing unnecessary wrapping divs from `icon` and `action` slots with no regressions

> given that styles for action slot are removed (https://github.com/stardust-ui/react/pull/1269/files?file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#diff-8bdbadd856993b56fb3e32cd80f44335L37), it seems questionable that classes.action originally applied to wrapper now is applied as styles to Button.
Could you, please, suggest the reasons for these changes?

styles for `action` slot are removed only for HC theme because they were redundant, we still apply styles for `action` slots in `packages/react/src/themes/teams/components/Attachment/attachmentStyles.ts`

And yes, it was probably wrong to apply them on the wrapper in initial implementation or the implementation is just outdated. Should be fine now.

The changes are done in order to optimize number of DOM nodes and apply styles to the correct element, hope it makes sense